paxos/proposer.hh

```diff
@@ -15,23 +15,34 @@ namespace witnesskvs::paxos {
 
 class Proposer {
  private:
+  uint64_t proposal_number_;
```

Comment:
Why is this field even needed ? The vector of bool should be enough right ?

Reply:
we need this to save proposal number after first one to use for subsequent accept request.